ckb-sync-mainnet/script/sync-mainnet.sh

```diff
@@ -183,8 +190,9 @@ function parse_report_and_inster_to_postgres() {
       speed=$(echo $LINE | awk -F '|' '{print $4}')
       tip=$(echo $LINE | awk -F '|' '{print $5}')
       hostname=$(echo $LINE | awk -F '|' '{print $6}')
-      psql -c "INSERT INTO sync_mainnet_report (github_run_id,time,ckb_version,ckb_commit_id,ckb_commit_time,time_s,speed,tip,hostname)  \
-             VALUES ('$GITHUB_RUN_ID','$time','$ckb_version','$CKB_COMMIT_ID','$CKB_COMMIT_TIME','$time_s','$speed','$tip','$hostname');"
+      replay_tps=$(echo $LINE | awk -F '|' '{print $8}')
```

Comment:
The replay_tps should contains  in sync-mainnet.brief.md.

Reply:
replay_tps already in sync-mainnet.brief.md, in [playbook.yml](https://github.com/nervosnetwork/ckb-integration-test/blob/8efd83c3175e8b9dfd392b1b2568d0ce1755e2fa/ckb-sync-mainnet/ansible/playbook.yml#L96)